dojo_plugin/api/v1/ssh_key.py

```diff
@@ -23,17 +25,18 @@ def post(self):
         key_value = data.get("ssh_key", "")
 
         if key_value:
-            key_re = "ssh-(rsa|ed25519|dss) AAAA[0-9A-Za-z+/]{1,730}[=]{0,2}"
-            key_match = re.match(key_re, key_value)
-            if not key_match:
+            try:
+                key = SSHKey(key_value, strict=True)
+                key.parse()
+                key_value = f"{key.key_type.decode()} {base64.b64encode(key._decoded_key).decode()}"
```

Comment:
noticed this in the `sshpubkeys` source:

```py
class SSHKey:
    # ...
  
    @property
    def key(self):
        """Base64 encoded key"""
        return base64.b64encode(self._decoded_key)
```

maybe this could do the same thing while being a less brittle against updates to the internals? `_` properities are usually not subject to semantic versioning